site/content/en/docs/api_sdk/sdk/highlevel-api.md

```diff
@@ -61,3 +62,107 @@ with make_client(host="http://localhost") as client:
     task.update({'name': 'mytask'})
     task.remove()
 ```
+
+## Client
+
+The `cvat_sdk.core.client.Client` class manages session, provides connection management and
+resource location operations. It is the staring point for using CVAT SDK.
+
+An instance of `Client` can be created directly by calling the class constructor,
+or with the utility function `cvat_sdk.core.client.make_client()`, which can handle
+some configuration for you. The extended configuration for `Client` is performed with
+the `cvat_sdk.core.client.Config` class instances. A `Config` object can be passed to
+the `Client` constructor and then it can be accessed with the `Client.config` field.
+`Client` objects implement [context manager protocol](https://docs.python.org/3/reference/datamodel.html#context-managers).
+
+You can create `Client` this way:
+
+```python
+from cvat_sdk import make_client
+
+with make_client('localhost', port='8080', credentials=(user, password)) as client:
+    ...
+```
+
```

Comment:
It would be useful here to provide an equivalent example using `Client`, so that the reader can understand what `make_client` does for you.

Reply:
I've added some explanations. I don't think it is a good idea to fix the actions done by this function in the documentation, because the code tends to change quickly.